### ISSUE TITLE
[a11y] All links on banners should be monochrome links

### ISSUE THREAD

**Issue by mirualves** [Shopify/polaris-react#1827]
## Issue summary
We have changed the color of the `Links` component, so it meets 4.5 color contrast ratio on all background colors when fixing [a11y] Banner link color and WCAG 2.1 link without underline contrast with text colour] #1620

All links in the banner component were replaced from `Default links` to `Monochrome link`, but we ended up leaving `External link` behind when dealing with the issue.

<img width="993" alt="Screenshot 2019-07-11 13 02 37" src="https://user-images.githubusercontent.com/5349023/61142029-cea0aa00-a49c-11e9-8586-dc17cdbff063.png">

## Expected behavior
All links on banners should have be monochrome links, external links as well.

**Comment by BPScott** [comment]
The work to fix this in #1651 was merged into the version-4.0.0 branch, because of it used context wanted to avoid introducing another clunky legacy context in v3.

From what I can see the links are always still blue in in the master branch, but are the correct monochrome in the version-4.0.0 branch, regardless of if they are external or not.

**Comment by mirualves** [comment]
Thanks for the info! Should I just close this issue?
